com.unity.netcode.gameobjects/Prototyping/NetworkTransform.cs

```diff
@@ -251,100 +374,123 @@ internal bool UpdateNetworkState(ref NetworkState networkState)
             {
-                rotAngles.y = networkState.RotAngleY;
+                interpolatedRotAngles.y = RotationInterpolator.GetInterpolatedValue().eulerAngles.y;
             }
+
             if (networkState.HasRotAngleZ)
             {
-                rotAngles.z = networkState.RotAngleZ;
+                interpolatedRotAngles.z = RotationInterpolator.GetInterpolatedValue().eulerAngles.z;
             }
+
             // Scale Read
             if (networkState.HasScaleX)
             {
-                scale.x = networkState.ScaleX;
+                interpolatedScale.x = ScaleXInterpolator.GetInterpolatedValue();
             }
+
             if (networkState.HasScaleY)
             {
-                scale.y = networkState.ScaleY;
+                interpolatedScale.y = ScaleYInterpolator.GetInterpolatedValue();
             }
+
             if (networkState.HasScaleZ)
             {
-                scale.z = networkState.ScaleZ;
+                interpolatedScale.z = ScaleZInterpolator.GetInterpolatedValue();
             }
 
             // Position Apply
             if (networkState.HasPositionX || networkState.HasPositionY || networkState.HasPositionZ)
             {
                 if (InLocalSpace)
                 {
-                    m_Transform.localPosition = position;
+                    m_Transform.localPosition = interpolatedPosition;
                 }
                 else
                 {
-                    m_Transform.position = position;
+                    m_Transform.position = interpolatedPosition;
                 }
+
+                PrevNetworkState.Position = interpolatedPosition;
             }
+
             // RotAngles Apply
             if (networkState.HasRotAngleX || networkState.HasRotAngleY || networkState.HasRotAngleZ)
             {
                 if (InLocalSpace)
                 {
-                    m_Transform.localEulerAngles = rotAngles;
+                    m_Transform.localRotation = Quaternion.Euler(interpolatedRotAngles);
                 }
                 else
                 {
-                    m_Transform.eulerAngles = rotAngles;
+                    m_Transform.rotation = Quaternion.Euler(interpolatedRotAngles);
                 }
+
+                PrevNetworkState.Rotation = interpolatedRotAngles;
             }
+
             // Scale Apply
             if (networkState.HasScaleX || networkState.HasScaleY || networkState.HasScaleZ)
             {
                 if (InLocalSpace)
                 {
-                    m_Transform.localScale = scale;
+                    m_Transform.localScale = interpolatedScale;
                 }
                 else
                 {
                     m_Transform.localScale = Vector3.one;
                     var lossyScale = m_Transform.lossyScale;
+                    // todo this conversion is messing with interpolation. local scale interpolates fine, lossy scale is jittery. must investigate
```

Comment:
Is this still relevant? Is it something we plan to solve (on the backlog)?

Reply:
MTT-1208